spec/draper/configuration_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+
+module Draper
+  RSpec.describe Configuration do
+    it 'yields Draper on configure' do
+      Draper.configure { |config| expect(config).to be Draper }
+    end
+
+    it 'defaults default_controller to ApplicationController' do
+      skip
```

Comment:
I was able to fix this issue. The problem was that one of the tests in the `build_strategy_spec.rb` was stubbing a constant that was overriding the new class definition in the `spec_helper.rb`. The specific test that was stubbing was able to be re-written without mocks now that the `ApplicationController` class is defined. I believe the way I re-wrote the test still tests for the same functionality. Somebody correct me if that's not the case. The commit is here: 
https://github.com/drapergem/draper/pull/788/commits/a886d288aea00c09388bc0fd6caac32badda6aaa